fake-cucumber/javascript/src/FakeTestResultsStream.ts

```diff
@@ -25,26 +26,8 @@ class FakeTestResultsStream extends Transform {
     this.p(envelope)
 
     if (envelope.source) {
-      const stepDefinitionPatterns = ['passed', 'failed', 'pending', 'skipped', 'ambig', 'ambiguous']
-      stepDefinitionPatterns.forEach(pattern => {
-        const stepDefinitionId = uuidv4()
-        this.stepDefinitionIds.push(stepDefinitionId)
-        this.p(
-          new messages.Envelope({
-            stepDefinitionConfig: new messages.StepDefinitionConfig({
-              id: stepDefinitionId,
-              pattern: new messages.StepDefinitionPattern({
-                source: `{}${pattern}{}`,
-              }),
-              location: new messages.SourceReference({
-                uri: "some/javascript/file.js",
-                location: new messages.Location({
-                  line: this.stepDefinitionIds.length * 3
-                })
-              })
-            })
-          })
-        )
+      this.stepDefinitions.makeMessages().forEach(message => {
```

Comment:
Adding the step definitions here is problematic, since it will happen for each feature file. We don't want to do this for each feature file - we only want to do it once.

Maybe it's better to pass all the step definitions to the stream's constructor.